test/types/fastify.test-d.ts

```diff
@@ -232,3 +233,6 @@ const ajvErrorObject: AjvErrorObject = {
   message: ''
 }
 expectAssignable<ValidationResult>(ajvErrorObject)
+
+const routeGeneric: RouteGenericInterface = {}
+expectType<RouteGenericInterface>(routeGeneric)
```

Comment:
Well, that is quite circular.

With this you test atleast the shape.
```suggestion
expectType<unknown>(routeGeneric.Body)
expectType<unknown>(routeGeneric.Headers)
expectType<unknown>(routeGeneric.Params)
expectType<unknown>(routeGeneric.Querystring)
expectType<unknown>(routeGeneric.Reply)
```